catalog/ui/catalog-ui-search/src/main/webapp/js/model/Query.js

```diff
@@ -497,7 +511,7 @@ define([
                 return _.pick(data, 'src', 'start', 'count', 'timeout', 'cql', 'sort', 'id');
             },
 
-            startSearch: function (forDeleted) {
+            startSearch: function (options) {
```

Comment:
Would also be a good idea to update the option parameter names to match the method names that will get called.  (limitToDeleted and limitToHistoric respectively).